graphrag/config/models/vector_store_config.py

```diff
@@ -85,9 +87,25 @@ def _validate_url(self) -> None:
         default=vector_store_defaults.overwrite,
     )
 
+    embeddings_schema: dict[str, VectorStoreSchemaConfig] = {}
+
+    def _validate_embeddings_schema(self) -> None:
+        """Validate the embeddings schema."""
+        for name in self.embeddings_schema:
+            if name not in all_embeddings:
+                msg = f"vector_store.embeddings_schema contains an invalid embedding schema name: {name}. Please rerun `graphrag init` and select the correct embedding schema names."
```

Comment:
You don't need to re-run init - you could just say to update their settings.yaml.